docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -17,32 +17,96 @@ keywords:
+- **Enhanced security and connectivity in Temporal Cloud**
+  - Built-in access controls. Restrict which callers can use a Nexus Endpoint.
+  - Global mTLS-secured Envoy mesh, connecting all Namespaces in an Account.
+
+**Should you be using Temporal Nexus?**
+
+- **Cross-team, cross-domain, and cross-namespace use cases** \-
+Nexus is purpose-built to connect Temporal Applications within and across Namespaces.
+It addresses the limitations of Child Workflows and Activity Wrappers that target a remote Namespace.
+Nexus has a streamlined Temporal developer experience, reliable execution, and integrated observability.
+
+- **Share a subset of a Temporal Application** \-
+Abstract and share a subset of an Application as a Nexus Service.
+Expose Services on a Nexus Endpoint for others to use and secure access with Endpoint access control policies.
+Nexus Endpoints decouple callers from handlers, so teams can operate more autonomously.
 
-For a deep dive into Temporal Nexus, see the [Temporal Nexus Encyclopedia section](https://docs.temporal.io/nexus).
-To connect with the Nexus community, join the [#nexus](https://temporalio.slack.com/archives/C07LQN0JK9B) channel in [Temporal Slack](https://t.mp/slack).
+- **Design for growth** \-
+Temporal Nexus enables a modular application design that can evolve as you grow.
+Start with Nexus Services in a monolithic Namespace and move Services to separate Namespaces with small configuration changes and no downtime for service consumers.
 
-Temporal keeps the runtime benefits of an Event-Driven Architecture (EDA) where services can be down without affecting each other, but offers a higher-level durable execution programming model that lets you write idiomatic code in any language and preserve the state of execution.
-This ensures that your Workflows can be resumed automatically in the presence of failure, without the extensive boilerplate code and complexity that comes with using raw EDA frameworks directly.
+- **Smaller failure domains** \- When teams operate in the same monolithic Namespace, everything is available to everyone, and mis-behaving workers can trigger rate limits that affect all teams operating in that monolithic Namespace.
+  Nexus enables each team to have their own Namespace for improved security, troubleshooting, and fault isolation.
 
-With Temporal Nexus, inter-service communication and durable execution can span teams, Namespaces, and regions all from within the Temporal context.
-Nexus preserves a familiar API programming model that supports both synchronous and asynchronous patterns, is suitable for low-latency and long-running operations, and uses an optimized form of the Temporal queue-based Worker architecture to preserve the benefits of EDA at runtime.
+- **Multi-region use cases** \-
+Nexus requests in Temporal Cloud are routed across a global mTLS-secured Envoy mesh.
+Reliable execution is provided with built-in Nexus Machinery and Temporal's queue-based Worker architecture.
```

Comment:
If you flip the sentence you can get rid of passive voice. 

Is the reliability the queue-based architecture or is it the centralized orchestration and dispatch of Workflow Executions to Worker fleets?

Reply:
reworded, thanks.